cmd/consumer/main.go

```diff
@@ -30,12 +30,13 @@ func main() {
 
 	sess := session.Must(session.NewSession(aws.NewConfig().WithRegion(getEnv("FIREHOSE_AWS_REGION")).WithMaxRetries(4)))
```

Comment:
why not use the default `AWS_REGION` which is automatically read by the aws-sdk?